diff --git a/recipes/gst-plugins-bad-1.0.recipe b/recipes/gst-plugins-bad-1.0.recipe
index 0421c48..972feef 100644
--- a/recipes/gst-plugins-bad-1.0.recipe
+++ b/recipes/gst-plugins-bad-1.0.recipe
@@ -341,7 +342,8 @@ class Recipe(custom.GStreamer):
     files_typelibs = [
         'GstInsertBin-1.0',
         'GstMpegts-1.0',
-        'GstPlayer-1.0'
+        'GstPlayer-1.0',
+        'GstWebRTC-1.0'
         ]
 
     platform_files_codecs_devel = {
