diff --git a/packages/gstreamer-1.0-codecs.package b/packages/gstreamer-1.0-codecs.package
index 61109520..f5cfe571 100644
--- a/packages/gstreamer-1.0-codecs.package
+++ b/packages/gstreamer-1.0-codecs.package
@@ -31,5 +31,6 @@ class Package(custom.GStreamer, package.Package):
 
     def prepare(self):
         if self.config.variants.rpi:
-            self.files.append('gst-omx-1.0:plugins_codecs')
-            self.files_devel.append('gst-omx-1.0:plugins_codecs_devel')
+            # self.files.append('gst-omx-1.0:plugins_codecs')
+            # self.files_devel.append('gst-omx-1.0:plugins_codecs_devel')
+            pass
