diff --git a/recipes/gst-plugins-bad-1.0.recipe b/recipes/gst-plugins-bad-1.0.recipe
index 01ab4cb..ce5ba50 100644
--- a/recipes/gst-plugins-bad-1.0.recipe
+++ b/recipes/gst-plugins-bad-1.0.recipe
@@ -11,7 +11,7 @@ class Recipe(custom.GStreamer):
                      'assrender': 'enabled', 'kate': 'enabled', 'openh264': 'enabled',
                      'opus': 'enabled', 'hls': 'enabled', 'rtmp': 'enabled',
                      'srtp': 'enabled', 'dts': 'enabled', 'libmms': 'enabled',
-                     'resindvd': 'enabled', 'soundtouch': 'enabled', 'openjpeg': 'enabled',
+                     'resindvd': 'enabled', 'openjpeg': 'enabled',
                      'voaacenc': 'enabled', 'rsvg': 'enabled', 'openglmixers': 'enabled',
                      'spandsp': 'enabled', 'decklink': 'enabled', 'webrtc': 'enabled',
                      'dtls': 'enabled', 'srt': 'enabled', 'webrtcdsp': 'enabled'}
@@ -19,7 +19,7 @@ class Recipe(custom.GStreamer):
     deps = ['gstreamer-1.0', 'gst-plugins-base-1.0', 'bzip2', 'libass',
             'libkate', 'zlib', 'openh264', 'opus', 'nettle', 'librtmp',
             'libsrtp', 'libdca', 'libmms', 'libdvdnav', 'libnice',
-            'soundtouch', 'vo-aacenc', 'librsvg', 'openjpeg', 'openssl',
+            'vo-aacenc', 'librsvg', 'openjpeg', 'openssl',
             'spandsp', 'webrtc-audio-processing', 'sbc', 'ladspa', 'srt']
     use_system_libs = True
 
