diff --git a/recipes/gst-plugins-bad-1.0.recipe b/recipes/gst-plugins-bad-1.0.recipe
index 0421c48..6cb3548 100644
--- a/recipes/gst-plugins-bad-1.0.recipe
+++ b/recipes/gst-plugins-bad-1.0.recipe
@@ -14,7 +14,8 @@ class Recipe(custom.GStreamer):
                      'resindvd': 'enabled', 'soundtouch': 'enabled', 'openjpeg': 'enabled',
                      'voaacenc': 'enabled', 'rsvg': 'enabled', 'openglmixers': 'enabled',
                      'spandsp': 'enabled', 'decklink': 'enabled', 'webrtc': 'enabled',
-                     'dtls': 'enabled', 'srt': 'enabled', 'webrtcdsp': 'enabled'}
+                     'dtls': 'enabled', 'srt': 'enabled', 'webrtcdsp': 'enabled',
+                     'introspection' : 'enabled' }
     remotes = {'origin': 'https://anongit.freedesktop.org/git/gstreamer/gst-plugins-bad'}
     deps = ['gstreamer-1.0', 'gst-plugins-base-1.0', 'bzip2', 'libass',
             'libkate', 'zlib', 'openh264', 'opus', 'nettle', 'librtmp',
